app/controllers/feedbacks_controller.rb

```diff
@@ -0,0 +1,36 @@
+class FeedbacksController < ApplicationController
+  before_action :set_feedback, only: %i[show edit update]
+
+  def index; end
+
+  def show
+    authorize @feedback
+  end
+
+  def new; end
+
+  def edit
+    authorize @feedback
+  end
+
+  def update
+    authorize @feedback
+
+    if @feedback.update_with_json_answers(params)
+      flash[:notice] = 'Feedback was successfully submitted.'
+      redirect_to feedback_path(@feedback)
+    else
+      render :edit, status: :unprocessable_entity
+    end
+  end
+
+  private
+
+  def set_feedback
+    @feedback = Feedback.find(params[:id])
+  end
+
+  def feedback_params
+    params.require(:feedback).permit!
```

Comment:
I'm pretty sure you can permit nested params?? Like an array of stuff.

Reply:
This was a mistake left in while I was experimenting with things. I added them back in.